mlrun/projects/operations.py

```diff
@@ -187,6 +187,11 @@ def my_pipe(url=""):
     task.spec.verbose = task.spec.verbose or verbose
 
     if engine == "kfp":
+        if schedule:
+            raise mlrun.errors.MLRunInvalidArgumentError(
+                "Schedule is not supported in KFP run function."
```

Comment:
"Scheduling is not supported in KFP run function."
                "Use either project.set_schedule() or project.run() to schedule runs."